.github/workflows/publish_alpha.yml

```diff
@@ -0,0 +1,122 @@
+# This workflow will generate a distribution and upload it to PyPI
+
+name: Publish Alpha Build ...aX
+on:
+  workflow_call:
+    inputs:
+      branch:
+        type: string
+        default: ${{ github.ref }}
+      version_file:
+        type: string
+        default: version.py
+      locale_folder:
+        type: string
+        default: "locale"
+
+jobs:
+  autotranslate:
+    runs-on: ubuntu-latest
+    env:
+      API_KEY: ${{secrets.DL_API_KEY}} 
+      LOCALE_FOLDER: ${{ github.workspace }}/action/package/${{ inputs.locale_folder }}
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.branch }}
+          path: action/package/
+      - name: Checkout Scripts Repo
+        uses: actions/checkout@v3
+        with:
+          repository: OpenVoiceOS/.github
+          path: action/github/
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          working-directory: action/package/
+          filters: |
+            locales:
+              - '${{ inputs.locale_folder }}/en-us/**'
+              - '${{ inputs.locale_folder }}/../dialog/en-us/**'
+              - '${{ inputs.locale_folder }}/../vocab/en-us/**'
+      - name: Setup Python
+        if: steps.filter.outputs.locales == 'true'
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
+      - name: Auto Translate
+        if: steps.filter.outputs.locales == 'true'
+        run: |
+          python -m pip install ovos-translate-plugin-deepl ovos-utils
+          python action/github/scripts/translate.py
+      - name: Commit to dev
+        if: steps.filter.outputs.locales == 'true'
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: autotranslate
+          repository: action/package/
```

Comment:
I wonder if there is a way to call another workflow, or use a fragment, so this code doesn't need to be maintained in multiple locations